plugins/tweet.py

```diff
@@ -3,31 +3,53 @@
 # By @Krishna_Singhal
 
 import os
-import re
 import requests
 
 from PIL import Image
 from validators.url import url
 
+from userge.utils import demojify
 from userge import userge, Config, Message
 
 CONVERTED_IMG = Config.DOWN_PATH + "img.png"
 CONVERTED_STIKR = Config.DOWN_PATH + "sticker.webp"
 
-EMOJI_PATTERN = re.compile(
-    "["
-    "\U0001F1E0-\U0001F1FF"  # flags (iOS)
-    "\U0001F300-\U0001F5FF"  # symbols & pictographs
-    "\U0001F600-\U0001F64F"  # emoticons
-    "\U0001F680-\U0001F6FF"  # transport & map symbols
-    "\U0001F700-\U0001F77F"  # alchemical symbols
-    "\U0001F780-\U0001F7FF"  # Geometric Shapes Extended
-    "\U0001F800-\U0001F8FF"  # Supplemental Arrows-C
-    "\U0001F900-\U0001F9FF"  # Supplemental Symbols and Pictographs
-    "\U0001FA00-\U0001FA6F"  # Chess Symbols
-    "\U0001FA70-\U0001FAFF"  # Symbols and Pictographs Extended-A
-    "\U00002702-\U000027B0"  # Dingbats
-    "]+")
+CELEBRITIES = {
```

Comment:
Addressed in #118  response from dev -
_"it's my plugin, my cmd, my idea, my telegraph link, 
why u interfaring 😂🤷‍♂"_

Reply:
plugin chor haye haye